src/Renderer/TrafficRenderer.cpp

```diff
@@ -59,6 +59,7 @@ TrafficRenderer::Draw(Canvas &canvas, const TrafficLook &traffic_look,
       canvas.Select(traffic_look.alarm_brush);
       break;
     case FlarmTraffic::AlarmType::NONE:
+      if(colorful_traffic){} // avoid unused variable warning
```

Comment:
What????

Reply:
I wanted to split the feature into two commits. here was the most natural point. what should I do about it?

Reply:
Why did you want to split the feature into two commits? The first commit is incomplete and thus doesn't make sense. Commits that don't make sense won't be merged.